src/components/Notification/index.js

```diff
@@ -0,0 +1,174 @@
+import './notification.scss'
+import { captalize } from '../../utils/captalize'
```

Comment:
The `captalize` name is a typing mistake. The correct is `capitalize`. You can import the correct function directly from `utils` directory ([like you can see here](https://github.com/storyblok/storyblok-design-system/blob/feature%2Fnotification-component/src/utils/index.js#L4)):

```js
import { capitalize } from '../../utils'
```

And, please remove the file with incorrect name.